src/main/Config/AppConfig.java

```diff
@@ -31,11 +35,21 @@ public static Javalin appFactory(DeploymentLevel deploymentLevel) {
     LogFactory l = new LogFactory();
     l.createLogger();
 
+    /* Create the Encryption Controller for encryption */
+    EncryptionController encryptionController;
+    try {
+      encryptionController = new EncryptionController(db);
+    } catch (GeneralSecurityException | IOException e) {
+      // Is this correct? #Connor
+      return null;
```

Comment:
no dont return null? why do we have to try catch the initialization of encryption controller? Initialization should ideally not have exceptions thrown

Reply:
It is because when we generate the encryption controller, we have to generate an AEAD key to be used for the whole code, and this throws an error